src/constants/properties.constant.ts

```diff
@@ -0,0 +1,37 @@
+const PROPERTIES = {
+  GLOBAL: {
```

Comment:
Can we use `export const PROPERTIES= ... {}`?
This to keep it consistent

Reply:
That format is already present on `main`, so I've raised [a separate PR](https://github.com/UK-Export-Finance/tfs-api/pull/30) to fix this comment